lib/experimental/Widgets/Content/TasksList/TaskItem/index.tsx

```diff
@@ -45,7 +45,7 @@ export function TaskItem({
   hideIcon = false,
 }: TaskItemProps) {
   const className = cn(
-    "flex flex-row items-center gap-1 rounded-sm border border-solid border-transparent px-2 py-1.5",
+    "flex flex-row items-center gap-1 rounded-sm border border-solid border-transparent px-2 py-1.5 text-f1-foreground",
```

Comment:
Needed to add this class because otherwise items show up with a link color

<img width="410" alt="Screenshot 2024-11-07 at 17 28 48" src="https://github.com/user-attachments/assets/95fe3819-b32f-42a8-99cf-178d937e9dcf">